serve/package.go

```diff
@@ -206,7 +208,7 @@ func copyFile(src, dst string) error {
 
 func (s *PluginServe) newCmdPluginPackage() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "package <plugin_directory> <version>",
+		Use:   "package <plugin_directory> <version> -m <message>",
```

Comment:
Ideally arguments should come later than parameters... `package -m <message> <dir> <version>`